### PR TITLE
Improvements to the automation based on challenge feedback

### PR DESCRIPTION
**Pull request by florianesser**
- improve documentation regarding file names and locations
- summary log: add validation type to entries to make them distinguishable
- summary log: add aggregation runs
- clean aggregation output folder before a rerun
- summary log: write to config base path instead of script folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,16 +169,18 @@ Für den Datenfluss 7_10 wird in der Standardkonfiguration folgende Ordnerstrukt
...
 
-Die Validierungen können über `gradlew validate-nap-<quelle>-<plan>` auch einzeln ausgeführt werden.
+Die Validierungen können über `gradlew validate-nap-<lärmquelle>-<plan>` auch einzeln ausgeführt werden.
 Eine Liste aller verfügbaren Validierungen kann mit `gradlew tasks` abgerufen werden.
 
 Bei jedem Validierungslauf wird eine Liste der durchgeführten Validierungen in der Datei `Zusammenfassung.log`
-angehängt, die das Ergebnis für jede validierte Eingabedatei zeigt.
+angehängt, die das Ergebnis für jede validierte Eingabedatei zeigt. Die Datei `Zusammenfassung.log` wird aus technischen
+Gründen im Hauptordner der Automatisierungsskripte angelegt und nicht in dem Ordner, der in der Konfigurationseinstellung
+`configBaseBath` angegeben ist.
```
[the remark this answers is not in the document]
Comment (florianesser) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -203,14 +205,14 @@ Aggregation wird ebenfalls Validierungsfehler aufweisen. Um Pläne mit Validieru
 muss das entsprechende Planverzeichnis aus dem Validierungsordner (z. B. `<configBasePath>/majorroad/validation`) entfernt
 oder von dort verschoben werden.
 
-Die Aggregationen können über `gradlew aggregate-nap-<quelle>` auch einzeln ausgeführt werden.
+Die Aggregationen können über `gradlew aggregate-nap-<lärmquelle>` auch einzeln ausgeführt werden.
 Eine Liste aller verfügbaren Aggregationen kann mit `gradlew tasks` abgerufen werden.
 
 Die detaillierten Log-Ausgaben und Berichte werden im Ausgabeverzeichnis der Quelle
```
[the remark this answers is not in the document]
Comment (florianesser) [reply README.md:216]: Added a paragraph

File: successEvaluation.groovy [hunk 3]
```diff
@@ -9,16 +9,25 @@ else {
 }
 def sourceFile = new File(sourceFileUrl.toURI())
 
+def validationType
+def targetSchema = aggregated['eu.esdihumboldt.hale.io.schema.read.target'].report.location
+if (targetSchema.toString().contains('CoverageArea')) {
+    validationType = 'Gültigkeitsbereich'
```
[the remark this answers is not in the document]
Comment (florianesser) [reply successEvaluation.groovy:46]: Added

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (florianesser) [reply README.md]: Fixed

**Review by annat2022 — APPROVED** [review]
LGTM

**Pull request merged**